rll/collectd.sh

```diff
@@ -158,34 +158,34 @@ EOF
 
 # Overwrite and backup the collectd configuration if it has changed
 if run_check_write_needed $collectd_conf $collectd_conf_tmp; then
-  chmod 0644 $collectd_conf_tmp
-  [[ -f $collectd_conf ]] && cp --archive $collectd_conf "${collectd_conf}.`date -u +%Y%m%d%H%M%S`"
-  mv --force $collectd_conf_tmp $collectd_conf
+  sudo chmod 0644 $collectd_conf_tmp
+  [[ -f $collectd_conf ]] && sudo cp --archive $collectd_conf "${collectd_conf}.`date -u +%Y%m%d%H%M%S`"
```

Comment:
Same issue as listed earlier, if we are sure `$collectd_conf` is readable by rightlink, then this is okay otherwise we should have this check with sudo.